### DISCUSSION TITLE
Elasticsearch as Connector or Service

### DISCUSSION THREAD

**Discussion by maehue** [open-metadata/OpenMetadata#12759] [Ideas]
I want to be able to use OpenMetadata for documenting Elasticsearch Indexes and their related fields and mappings. I don't think this is available currently, though I know Elasticsearch is part of the OpenMetadata architecture. 

I'm curious is there some way to do this already or would it be new feature?

**Comment by lukens** [comment]
It seems like maybe Elasticsearch was supported at one point, but has maybe since been dropped?

In the video on the homepage, it is listed in the Integrations:

<img width="909" alt="Screenshot 2023-10-05 at 13 42 06" src="https://github.com/open-metadata/OpenMetadata/assets/686128/94b56d0b-410e-4aa9-8275-279927e32d96">


And if you try searching the documentation, you get results matching Elasticsearch, but you get a 404 if you try and visit the pages.

<img width="655" alt="Screenshot 2023-10-05 at 13 42 58" src="https://github.com/open-metadata/OpenMetadata/assets/686128/117f5174-6f40-4e21-96de-4c08ea63e232">


I don't know if anyone involved with the project can speak to this?

**Comment by ulixius9** [comment]
Hey @lukens & @maehue, we have finished the development of elasticsearch as a connector in OpenMetadata (ref [pr](https://github.com/open-metadata/OpenMetadata/pull/12892)), and this will be available with our upcoming version 1.2.0.
